.github/workflows/unit_testing.yml

```diff
@@ -1,20 +1,34 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
 name: Unit Test Workflow
+
 on:
   push:
-    branches:
-      - dev
-      - main
+    branches: [ "dev" ]
+  pull_request:
+    branches: [ "dev" ]
+
+env:
+    WORKING_DIRECTORY: ./
+
 jobs:
   build:
+
     runs-on: ubuntu-latest
+
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x, 20.x]
+        node-version: [14.x, 16.x, 18.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
     steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ matrix.node-version }}
-      - run: npm install
-      - run: npm run unit
\ No newline at end of file
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm -w buefy-next ci --prefix "${{ env.WORKING_DIRECTORY }}"
+    - run: npm -w buefy-next run build --prefix "${{ env.WORKING_DIRECTORY }}"
+    - run: npm -w buefy-next test --prefix "${{ env.WORKING_DIRECTORY }}"
```

Comment:
As far as I tested, `-w buefy-next` did not work but `-w @ntohq/buefy-next` did. You do not have to run `build` before tests. And I do not think `--prefix` options are necessary. So the above three lines will become:
```lml
- run: npm ci
- run: npm -w @ntohq/buefy-next test
```

Reply:
I forgot that you changed the name of the package. Therefore to access the buefy-next I must use `-w @ntohq/buefy-next`